src/main/java/bio/terra/stairway/FlightDao.java

```diff
@@ -389,6 +395,7 @@ private static Exception getExceptionFromJson(String exceptionJson) {
             }
             return getObjectMapper().readValue(exceptionJson, Exception.class);
         } catch (IOException ex) {
+            logger.error("JACKSON deserialization failed. JSON:" + exceptionJson, ex);
```

Comment:
Do we need this logging? I have been told that the logging in the global exception handler would be sufficient.
If it isn't sufficient (and from looking at stackdriver logs, I'm not sure), then we should either make it so or follow this (anti-)pattern so we get good stack dumps.

Reply:
In my [first PR for this ticket](https://github.com/DataBiosphere/jade-data-repo/pull/324), I didn't include this exception for exactly the reason you say, and then it was requested that I include it, so I did so here. My preference is the same as before, not to include it, but I also don't feel strongly about it -- whatever the group prefers is fine.

Relevant comments from previous PR:
jh: If you pass in ex as a second positional argument, it will print the stack trace in the log :)

dd: Is that always true? I have had difficulty finding stack trace in the stackdriver logs, but that may just be some lack of skill on my part.

mm: The ex variable here is the JACKSON failure to deserialize JSON exception, not the actual underlying cause -- that's in the exceptionJson variable. The throw on the next line includes the JACKSON exception. That's the reason I didn't include ex in the logger call -- because I thought it would be redundant. But maybe it's good to keep it together or I'm mistaken about the exception eventually being logged higher up? What do you think?

jh: I'd prefer to err on the side of more info than less in the logs at least for now, since we are missing stacktraces in a few places. @ddietterich yes this is always the case for logger.error. I am unsure if it works for the other log levels but I would assume so.

Reply:
Or maybe I'm misunderstanding you -- do you mean to remove the logger statement completely or just the IOException? I think we do want the logger statement for the problematic exceptionJSON, since that's not included in the IOException thrown by Jackson. But I don't think we need the IOException included in the logger statement. Sorry for any confusion!

Reply:
It is fine as is.